lib/src/base-styles/_nx-lists.scss

```diff
@@ -70,12 +70,15 @@
   grid-area: list-text;
   margin: 0;
   padding-right: var(--nx-spacing-6x);
+  max-width: var(--nx-width-text-wrapping);
+
 
   &.nx-truncate-ellipsis {
     // Something about the grid layout gets confused due to overflow: hidden on this element
     // and adds vertical space to the second row of the grid.
     // Experimentally, setting an explicit height here fixes that.
     height: calc(var(--nx-line-height) * 1em);
+    max-width: unset;
   }
 }
 
```

Comment:
Should we add the max-width to `.nx-list__description` as well?

Reply:
> Should we add the max-width to `.nx-list__description` as well?

I think so: https://github.com/sonatype/sonatype-react-shared-components/pull/432/commits/c7ceeebcde0087f3015b66cb27125a666f3cefdb

Reply:
> I debated adding it to the terms as well (see note re CYA below) but in the end I did not.

On way or another terms should match form labels since that's the whole intent with `nx-read-only`

Reply:
> > I debated adding it to the terms as well (see note re CYA below) but in the end I did not.
> 
> One way or another terms should match form labels since that's the whole intent with `nx-read-only`

The terms have `width: 250px` on them so I think they're good.

Reply:
> The terms have width: 250px on them so I think they're good.

Ah right I was thinking of `nx-read-only` when I wrote that.